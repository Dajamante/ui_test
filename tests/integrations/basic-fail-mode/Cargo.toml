[package]
name = "basic_fail_mode"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dev-dependencies]
ui_test = { path = "../../.."}

[[test]]
name = "ui_tests"
harness = false

[[test]]
name = "run_file"
harness = true
