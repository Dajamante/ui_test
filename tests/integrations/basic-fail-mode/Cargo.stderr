    Finished test [unoptimized + debuginfo] target(s) 
     Running unittests src/lib.rs 
     Running tests/run_file.rs 
     Running tests/ui_tests.rs 
   Compiler flags: ["--error-format=json", "--edition=2021"]
   Building test dependencies...
tests/actual_tests/foomp.rs ... ok

test result: ok. 1 tests passed, 0 ignored, 0 filtered out

   Doc-tests basic_fail_mode
